spec/sidekiq/form1095/new1095_bs_job_spec.rb

```diff
@@ -51,68 +51,94 @@
       allow(object).to receive(:get).and_return(nil)
     end
 
-    it 'saves valid form from S3 file' do
```

Comment:
Github has completely mangled the diff here. Because this code change is date-specific, I added two top-level contexts: one for past tax years and one for the current tax year. Adding that context block changed the indentation for every line, which is why the diff is so large.